### ISSUE TITLE
Authentication card is not visible  in fastify-api-reference

### ISSUE THREAD

**Issue by alzalabany** [scalar/scalar#839]
### Expected Behavior

Global auth card should be visible under intro page

### Actual Behavior

its not there

### Reproduction steps

```bash
example specs:


{"openapi":"3.0.3","info":{"title":"CRMBox Api","version":"1.0.0"},"components":{"securitySchemes":{"bearerAuth":{"type":"http","scheme":"bearer","bearerFormat":"JWT"}},"schemas":{}},"paths":{"/auth/token/refresh":{"post":{"tags":["auth"],"responses":{"200":{"description":"Default Response"}}}},"/auth/token":{"post":{"tags":["auth"],"description":"exchange u/p for a 1hr token","requestBody":{"content":{"application/json":{"schema":{"type":"object","properties":{"username":{"type":"string"},"password":{"type":"string"}},"required":["username","password"],"additionalProperties":false}}},"required":true},"parameters":[{"schema":{},"in":"query","name":"not"},{"schema":"object","in":"query","name":"type"},{"schema":{"type":"string"},"in":"query","name":"additionalProperties"}],"responses":{"200":{"description":"Default Response","content":{"application/json":{"schema":{"type":"object","properties":{"access_token":{"type":"string"},"refresh_token":{"type":"string"}},"required":["access_token","refresh_token"],"additionalProperties":false}}}}}}},"/auth/token/email":{"post":{"tags":["auth"],"requestBody":{"content":{"application/json":{"schema":{"type":"object","properties":{"email":{"type":"string","format":"email"},"callbackURL":{"type":"string","format":"uri"}},"required":["email","callbackURL"],"additionalProperties":false}}},"required":true},"parameters":[{"schema":{},"in":"query","name":"not"},{"schema":"object","in":"query","name":"type"},{"schema":{"type":"string"},"in":"query","name":"additionalProperties"}],"responses":{"200":{"description":"Default Response","content":{"application/json":{"schema":{"not":{}}}}}}}},"/auth/token/email/{token}":{"get":{"tags":["auth"],"description":"exchange magic link for a 1min token. then redirect to callbackURL; callback should use this short-lived token to sign-in using /auth/token/refresh","parameters":[{"schema":{"type":"string"},"in":"path","name":"token","required":true}],"responses":{"200":{"description":"Default Response","content":{"application/json":{"schema":{"not":{}}}}}}}},"/admin":{"get":{"security":[{"bearerAuth":[]}],"responses":{"200":{"description":"Default Response","content":{"application/json":{"schema":{"type":"object","properties":{"hello":{"type":"string"}},"required":["hello"],"additionalProperties":false}}}}}}},"/config":{"post":{"tags":["config"],"requestBody":{"content":{"application/json":{"schema":{"type":"object","properties":{"configName":{"type":"string"},"configData":{},"encrypted":{"type":"boolean"}},"required":["configName","encrypted"],"additionalProperties":false}}},"required":true},"security":[{"bearerAuth":[]}],"responses":{"200":{"description":"Default Response","content":{"application/json":{"schema":{"type":"object","properties":{"configName":{"type":"string"},"configData":{},"encrypted":{"type":"boolean"},"createdAt":{"anyOf":[{"type":"string"},{"type":"string","format":"date-time"}]}},"required":["configName","encrypted","createdAt"],"additionalProperties":false}}}}}}},"/config/{name}":{"get":{"tags":["config"],"security":[{"bearerAuth":[]}],"parameters":[{"schema":{"type":"string"},"in":"path","name":"name","required":true}],"responses":{"200":{"description":"Default Response","content":{"application/json":{"schema":{"type":"object","properties":{"configName":{"type":"string"},"configData":{},"encrypted":{"type":"boolean"},"createdAt":{"anyOf":[{"type":"string"},{"type":"string","format":"date-time"}]}},"required":["configName","encrypted","createdAt"],"additionalProperties":false}}}},"400":{"description":"Default Response","content":{"application/json":{"schema":{"type":"object","properties":{"message":{"type":"string"}},"required":["message"],"additionalProperties":false}}}}}}},"/configs":{"get":{"tags":["config"],"security":[{"bearerAuth":[]}],"responses":{"200":{"description":"Default Response","content":{"application/json":{"schema":{"type":"array","items":{"type":"object","properties":{"configName":{"type":"string"},"configData":{},"encrypted":{"type":"boolean"},"createdAt":{"anyOf":[{"type":"string"},{"type":"string","format":"date-time"}]}},"required":["configName","encrypted","createdAt"],"additionalProperties":false}}}}}}}}},"servers":[],"security":[]}
```
```


### Link to site / OpenAPI Spec

_No response_

### Version

1.13.0

### What browsers are you seeing the problem on?

_No response_

**Comment by hanspagel** [comment]
Wait, it’s visible here. Need to investigate.

https://sandbox.scalar.com/e/Ojam_#tag/auth

**Comment by alzalabany** [comment]
another note, is that exact same schema will look like this in swagger-ui
(notice /auth endpoints are public)

![image](https://github.com/scalar/scalar/assets/3620349/da987a4b-4f1e-4a3b-a7c4-638b27dfff30)

but with scalar. all endpoints will show as-if they require auth header (screen shot from https://sandbox.scalar.com/e/Ojam_#tag/auth)

![image](https://github.com/scalar/scalar/assets/3620349/3b985e01-34a5-468e-abfc-f9af37dc2dc8)

**Comment by arthurfiorette** [comment]
We're still having this issue...


```json
{
  "components": {
    "schemas": { ... },
    "securitySchemes": {
      "bearerAuth": {
        "bearerFormat": "JWT",
        "scheme": "bearer",
        "type": "http"
      }
    }
  },
  "info": {
    "contact": { ... },
    "description": "A complete REST API for corral management",
    "license": { ... },
    "title": "Feed Manager",
    "version": "latest"
  },
  "openapi": "3.1.0",
  "paths": { ... },
  "servers": [ ... ]
}
```

![image](https://github.com/scalar/scalar/assets/47537704/ba03f9d1-f717-482c-b24f-d77bb5de224c)

```ts
await fastify.register(require("@scalar/fastify-api-reference"), { routePrefix: "/reference" })
```

```json
 "dependencies": {
    "@fastify/swagger": "^8.14.0",
    "@scalar/fastify-api-reference": "^1.17.3",
  },
```

**Comment by hanspagel** [comment]
That was a great hint! For whatever reason the fastify theme had CSS to hide the box. Fixed in #1066
